core/test/financial-templates/PricelessPositionManager.js

```diff
@@ -46,6 +46,8 @@ contract("PricelessPositionManager", function(accounts) {
   const expirationTimestamp = startTimestamp + 10000;
   const priceFeedIdentifier = web3.utils.utf8ToHex("UMATEST");
   const minSponsorTokens = "5";
+  // Set to zero places no restriction on withdrawal requests for the majoirty of test cases.
+  const gcrWithdrawalScalingFactor = toWei("0");
```

Comment:
nit: it's a bit weird to have the default value to 0 because a scaling factor of 0 implies some really strange outcome or multiplication by 0, but practically checking for 0 and returning early in `_checkCollateralizationWithdrawalRequest ` is probably easier than running the whole check with scaling factor set to 1 by default.

Reply:
ye, I agree. I originally implemented this such that if it was 1 the contracts would behave as they did before and all logic would continue without this check. However, it's this implementation ended up making less sense as a value of 1 does not result in the expected behavior without having other hard coded checks.